freebios/Makefile

```diff
@@ -0,0 +1,17 @@
+TC_PREFIX  = /home/exophase/pandora-dev
+PREFIX     = $(TC_PREFIX)/arm-2011.03
+AS         = $(PREFIX)/bin/arm-none-linux-gnueabi-gcc
+OBJCOPY    = $(PREFIX)/bin/arm-none-linux-gnueabi-objcopy
```

Comment:
This should probably be changed so you can actually build it with any newer arm crossbuilding toolchain

Reply:
The source code and Makefile come directly from the original source. I just converted the blobs into the byte arrays there. I guess you're right we could try rebuilding the BIOS files and updating the Makefile. I'll work on it in the next days.